dev-util/netbeans/netbeans_bin-19.recipe

```diff
@@ -0,0 +1,76 @@
+SUMMARY="An IDE for Java"
+DESCRIPTION="NetBeans IDE lets you quickly and easily develop Java desktop, \
+mobile, and web applications, as well as HTML5 applications with HTML, \
+JavaScript, and CSS. The IDE also provides a great set of tools for PHP and \
+C/C++ developers.
+It is free and open source and has a large community of users and developers \
+around the world."
+HOMEPAGE="https://netbeans.org/"
+COPYRIGHT="1997,2023 Oracle and/or its affiliates"
+LICENSE="Apache v2"
+REVISION="1"
+SOURCE_URI="https://dlcdn.apache.org/netbeans/netbeans/$portVersion/netbeans-$portVersion-bin.zip#noarchive"
+CHECKSUM_SHA256="8df70edd6307d08af5f957e96616917329132284e6c40e432bc64ef28acfd498"
+ADDITIONAL_FILES="netbeans-ide.rdef.in"
+
+ARCHITECTURES="any"
+DISABLE_SOURCE_PACKAGE="true"
+
+PROVIDES="
+	netbeans_bin = $portVersion
+	app:netbeans = $portVersion
+	"
+REQUIRES="
+	haiku
+	java:environment == 17
+	"
+
+BUILD_PREREQUIRES="
+	cmd:find
+	cmd:unzip
+	cmd:xargs
+	"
+
+INSTALL()
+{
+	mkdir -p $appsDir
+	cd $appsDir
+	unzip $sourceDir/netbeans-$portVersion-bin.zip
+
+	cd netbeans
+	rm -rf profiler {cnd,dlight,ide}/bin \
+		platform/modules/lib/{amd64,i386,x86}
+	find . -name "*.dll" -o -name "*.exe" -o -name "*.cmd" \
+		-o -name "*.bat" -o -name ".lastModified" | xargs rm
+
+	sed -i 's,\${HOME}/.netbeans,\${HOME}/config/settings/netbeans,g' bin/netbeans
+	sed -i 's,\${HOME}/.cache/netbeans,\${HOME}/config/settings/cache/netbeans,g' bin/netbeans
+
+	cat << EOF > $appsDir/netbeans/netbeans.sh
```

Comment:
This was here from previous version, netbeans can be launched from Tracker/Deskbar, so not sure we need to add the launchscript also?

Reply:
This sets the `PATH` so that the openjdk tools are part of it. I guess it makes sense to have these. Alternatively, one could install `openjdk17_default` to get the same effect with the default `PATH`.

Reply:
Let's keep it around then :)